services/checks/checks_test.go

```diff
@@ -421,20 +421,20 @@ func TestMustParseVersion(t *testing.T) {
 		expected, err := version.Parse("2.6.0")
 		require.NoError(t, err)
 
-		actual := mustParseVersion("2.6.0")
+		actual := version.MustParse("2.6.0")
 		assert.Equal(t, expected, actual)
 	})
 
 	t.Run("empty string", func(t *testing.T) {
 		f := func() {
-			mustParseVersion("")
+			version.MustParse("")
 		}
 		assert.Panics(t, f)
 	})
 
 	t.Run("invalid version", func(t *testing.T) {
 		f := func() {
-			mustParseVersion("invalid version")
+			version.MustParse("invalid version")
```

Comment:
Well, here we test a function that is from the different repository and already tested. Probably we can remove it.

Reply:
Actually, the whole TestMustParseVersion is not needed now :)